app/views/pages/index.html.erb

```diff
@@ -6,9 +6,10 @@
     <hr>
     <%= current_user.username %><br>
     <%= current_user.email %>
-    <%= link_to 'log out', session_path, method: :delete %>
+    <%= link_to 'Log Out', session_path, method: :delete %>
+    <%= link_to 'Edit Profile', edit_user_path(current_user) %>
   <% else %>
-    <h2>Welcome, stranger!</h2><br>
+    <h2>Welcome, stranger!</h2>
       <div class="row">
```

Comment:
Minor indentation note:
In html, if your code following an element that is self closing, or has a closing tag. 
it is clearer to indicat a sibling (not child) element by having it at the same indentation.

```
<h2>Welcome, Stranger!</h2>
<div>
...
</div>
```